docs/src/main/sphinx/connector/hive.rst

```diff
@@ -521,6 +521,36 @@ Property Name                                                   Description
 
 ``hive.metastore.thrift.client.ssl.trust-certificate-password`` Password for the trust store
 
+``hive.metastore.thrift.delete-files-on-drop``                  When a table is dropped, delete the underlying files
+                                                                corresponding to the table. Hive metastore is responsible
+                                                                of deleting the files of a table when it is dropped.
+                                                                This setting can be considered as a fallback in the
+                                                                case when the Hive metastore completed the drop operation
+                                                                without removing the files of the table. Default is ``false``.
+
+``hive.metastore.thrift.assume-canonical-partition-keys``       Enable optimized metastore partition fetching for non-string
```

Comment:
Fixes https://github.com/trinodb/trino/issues/6437

Reply:
Currently it just says "if you want faster, enable this", which isn't the whole truth
The documentation should say when this is ok to set and when it's not.

Reply:
@findepi  can you help me come up with a better description  here?

Reply:
Maybe 

Allow the metastore to assume that partition keys are normalized for non-string partition keys. This can lead to performance improvements for affected tables. Not that ..

Reply:
Changed to 

> Allow the metastore to assume that the values of partition
> columns can be converted to string values. This can lead to
> performance improvements in the queries which apply filters
> on the partition columns.
> Note that the partition keys of type ``timestamp`` do
> not get canonicalized. Default is ``false``.